debezium-core/src/main/java/io/debezium/schema/DatabaseSchema.java

```diff
@@ -15,6 +15,8 @@
  */
 public interface DatabaseSchema<I extends DataCollectionId> {
 
+    String NO_CAPTURED_DATA_COLLECTIONS_WARNING = "After applying include/exclude list filters there's nothing selected to capture changes from, please check your configuration";
```

Comment:
@Naros is the wording okay?

Reply:
I think the following sounds more appropriate.  Wdyt?
```suggestion
    String NO_CAPTURED_DATA_COLLECTIONS_WARNING = "After applying the include/exclude list filters, no changes will be captured.  Please check your configuration.";
```

Reply:
:man_shrugging: you are the pro!

Reply:
Done